src/pages/en/guides/cms/storyblok.mdx

```diff
@@ -368,29 +366,54 @@ export default defineConfig({
 
 ### Generating pages
 
+To create a route for a specific `page`, you can fetch its directly from the Storyblok API and pass it to the `Page` component. Create an `index.astro` file in `src/pages` to render the `home` page:
```

Comment:
```suggestion
To create a route for a specific `page`, you can fetch its content directly from the Storyblok API and pass it to the `Page` component. Create an `index.astro` file in `src/pages/` to render the `home` page:
```
Fetch "it", or fetch "its....(missing word/property)"? (I'd lean towards the latter, because you have to pass an actual *thing* to the `<Page />` component.

(This suggestion only edits what's there, and does not take into account Kevin's comment that perhaps the step should itself be a different instruction. 😄 )